### ISSUE TITLE
Feature request: option to disable passport animation for quick save/load like TR4+

### ISSUE THREAD

**Issue by aredfan** [LostArtefacts/TRX#1897]
This feature was requested by a member on the Lost Artefacts server earlier this year. Logging it so it doesn't get lost.

**Update:** This is a video of how saving/loading looks in TR4.  The quick save/load menu is instant without the delay which accompanies the passport animations.

https://github.com/user-attachments/assets/ae21b830-1bf4-4b58-ad95-12ac40fe2271

**Comment by Richard-L** [comment]
Afaik you can now instantly save and load via the console, right?

Maybe instead of this, would it be an option to just allow keybinds for:

- Load current save slot
- Save to current save slot
- Move to next save slot
- Move to previous save slot

Would be even quicker for them, and wouldn't mess up the game too much.

**Comment by aredfan** [comment]
> Afaik you can now instantly save and load via the console, right?
> 
> Maybe instead of this, would it be an option to just allow keybinds for:
> 
> * Load current save slot
> * Save to current save slot
> * Move to next save slot
> * Move to previous save slot
> 
> Would be even quicker for them, and wouldn't mess up the game too much.

~~I think your suggestion is already covered under #600.~~

**Edit:** scratch that, I reread the issue and it's not related.

The dev console isn't really a substitute for normal gameplay since it's more of a debugging tool.  New features will always carry a risk of introducing regressions but we do a good job of catching them.

**Comment by walkawayy** [comment]
Would this be accessing the full savegame list like you're in the passport without actually going through the trouble of waiting for the passport to open?

Other games also have something called quick-saves you can easily use from a menu by holding down a button to either quick save or load. I was thinking about maybe making a separate issue for it depending on what people think.

Dishonored 2:

![kpyeqc9qq18a1](https://github.com/user-attachments/assets/bf49313e-7960-4762-86f4-87a6e53fd6bf)

Baldur's Gate 3:

![image](https://github.com/user-attachments/assets/b5f45418-bce1-49ab-89ae-1c43ce29f0c6)


The only difference however is these games autosort savegames based on time rather than based on slots like classic TR. So I'm not sure how this would work unless we had a dedicated quick-save/load slot. However, that would limit the player to a single quick-save. 🤷‍♂️

**Comment by aredfan** [comment]
> Would this be accessing the full savegame list like you're in the passport without actually going through the trouble of waiting for the passport to open?

Yes that's correct, this is a video of how saving/loading looks in TR4.  The quick save/load menu is instant without the delay which accompanies the passport animations.

https://github.com/user-attachments/assets/ae21b830-1bf4-4b58-ad95-12ac40fe2271

**Sidenote:** Dishonored is such as great game.

**Comment by Richard-L** [comment]
Just for reference:

Duckstation quick save/load menu:

https://github.com/user-attachments/assets/fb0373c1-b269-4776-b13c-366c91d9d9ea

OpenLara (via 5 and 9 keys)

https://github.com/user-attachments/assets/51df2ce7-03a5-4302-a707-43a536938ac9
